sdk/test-utils/recorder/test/utils.spec.ts

```diff
@@ -178,56 +178,185 @@ ultramarine.com/url/PUBLIC
   });
 
   describe("filter secrets from content", () => {
+    function verifyFilterFunctionForJson(
```

Comment:
I think this function adds more lines than the lines it's supposed to remove. I could be wrong!